src/components/List.js

```diff
@@ -3,6 +3,71 @@ import { NavLink } from 'react-router-dom';
 import styles from '../List.module.css';
 import Details from './Details';
 import { updatePurchaseDate, deleteItem } from '../lib/firebase.js';
+import { withStyles } from '@material-ui/core/styles';
+import { green, orange, red, grey } from '@material-ui/core/colors';
+import {
+  FormControlLabel,
+  Checkbox,
+  IconButton,
+  Grid,
+  Paper,
+  Typography,
+} from '@material-ui/core';
+import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
+import DeleteOutlineIcon from '@material-ui/icons/DeleteOutline';
+import CircleUnchecked from '@material-ui/icons/RadioButtonUnchecked';
+import CircleCheckedFilled from '@material-ui/icons/CheckCircle';
+
+const DecoratedCheckbox = p => {
```

Comment:
This is awesome! Such a great way to define the base styles for the base decorated checkbox.